dsf-gdb/org.eclipse.cdt.dsf.gdb/src/org/eclipse/cdt/dsf/gdb/launching/FinalLaunchSequence.java

```diff
@@ -656,22 +662,14 @@ public void stepAttachToProcess(final RequestMonitor requestMonitor) {
 	 * If we are dealing with an remote attach session, perform the attach to debugger.
 	 * Bug 528145
 	 * @since 6.6
+	 *
+	 * When removing this method, do inline method refactoring
 	 */
-	@Execute
```

Comment:
probably we should keep Execute annotation, WDYS?

Reply:
yes